WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -116,7 +116,7 @@ def post_announcement
     end
     title = "#{comp.name} on #{date_range_str} in #{comp.cityName}, #{comp.countryId}"
 
-    body = "The [#{comp.name}](#{root_url}results/c.php?i=#{comp.id})"
+    body = "The [#{comp.name}](#{root_url(locale:nil)}results/c.php?i=#{comp.id})"
```

Comment:
I fear it wouldn't work to localize this link, though, because this is a post that's show to the whole world. I proposed a js based solution [here](https://github.com/cubing/worldcubeassociation.org/pull/460/#issuecomment-217570960).